src/hooks/useMonitoringEventCallback.ts

```diff
@@ -42,7 +42,7 @@ function useMonitoringEventCallback() {
       try {
         push(ref(db, 'trm'), {
           chainId,
-          origin: location.origin,
+          origin: window.location.origin,
```

Comment:
I like that this is now scoped to `window`.

Reply:
yeah, linter asked for it